backend/keystore/keystore.go

```diff
@@ -87,16 +88,26 @@ type Keystore interface {
 
 	// CanSignMessage returns true if the keystore can sign a message for a coin.
 	CanSignMessage(coin.Code) bool
+
 	// SignBTCMessage signs the message using the private key at the keypath. The scriptType is
 	// required to compute and verify the address. The returned signature is a 65 byte signature in
 	// Electrum format.
 	SignBTCMessage(message []byte, keypath signing.AbsoluteKeypath, scriptType signing.ScriptType) ([]byte, error)
+
 	// SignETHMessage signs the message using thep rivate key at the keypath. The result contains a
 	// 65 byte signature. The first 64 bytes are the secp256k1 signature in / compact format (R and
-	// S values), and the last byte is the recoverable id (recid).
+	// S values), and the last byte is the recoverable id (recid). Returns ErrSigningAborted if the user aborts.
 	SignETHMessage(message []byte, keypath signing.AbsoluteKeypath) ([]byte, error)
 
+	// ETHSignTypedMessage signs an Ethereum EIP-612 typed message. 27 is added to the recID to denote
```

Comment:
it's EIP-712.

you can also copy the same for above for clarity:

```
         // The result contains a
	// 65 byte signature. The first 64 bytes are the secp256k1 signature in / compact format (R and
	// S values), and the last byte is the recoverable id (recid).
```

vice versa, you can add `27 is added to the recID to denote an uncompressed pubkey.` to the doc of `SignETMessage()`, otherwise it looks like one is doing it and the other is not.